lib/main_common.pm

```diff
@@ -1376,7 +1376,6 @@ sub load_x11tests {
                 loadtest(is_sle('<15') ? "x11/rhythmbox" : "x11/gnome_music");
                 loadtest "x11/wireshark";
                 loadtest "x11/ImageMagick";
-                loadtest "x11/remote_desktop/screensharing_available" if is_sle("15-sp4+");
```

Comment:
Is it safe to remove this line? please double confirm there is no test which loads this module.